### PR TITLE
Start Cucumber features.

### PR DESCRIPTION
**Pull request by hermannloose**
Basic Cucumber setup, with two semi-meaningful features for orientation. Could probably be DRYer, but I did my best to use `pickle` wherever applicable, while bearing in mind that acceptance test [should not so much be about the elementary user interactions](http://aslakhellesoy.com/post/11055981222/the-training-wheels-came-off).

Still relies directly on the model for a lot of the _Then_ steps, which should be improved in the future.

### CODE REVIEW

**Comment by hermannloose** [comment]
@jupp0r Please review.

Actually, this might have already notified you, seeing that GitHub pulled in your comments from yesterday. They're awesome like that.

File: features/support/factory_girl.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+require 'factory_girl_rails'
```
Comment (jupp0r) [inline features/support/factory_girl.rb:1]: how is this related to cucumber?

File: features/admin_manages_rotations.feature [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  Background: I am logged in as an admin
+    Given I am logged in as an admin
+
+  @wip
+  Scenario Outline: adding a rotation
+    When I add a rotation with name: "<name>"
+    Then a rotation should exist with name: "<name>"
+
+    Examples:
```
Comment (jupp0r) [inline features/admin_manages_rotations.feature:15]: Seems like a lot of duplication between here and admin_manages_escalation_policies.feature. Why does the feature need 3 those cases?

File: features/support/factory_girl.rb (hunk 1 above)
Comment (hermannloose) [inline features/support/factory_girl.rb:1]: The factories are used for pickle's _Given a $model exists with $fields_ steps and so on, see comment below. It falls back to ActiveRecord, but that doesn't handle things like wanting to have for example a vanilla contact detail for a scenario, which would depend on a user that we don't want to explicitly create all the time. The factories create default values for their associations (using other factories) unless specified otherwise.

File: features/support/pickle.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Make sure that you are loading your factory of choice in your cucumber environment
+#
+# For machinist add: features/support/machinist.rb
+#
+#    require 'machinist/active_record' # or your chosen adaptor
+#    require File.dirname(__FILE__) + '/../../spec/blueprints' # or wherever your blueprints are
+#    Before { Sham.reset } # to reset Sham's seed between scenarios so each run has same random sequences
+#
+# For FactoryGirl add: features/support/factory_girl.rb
```
Comment (hermannloose) [inline features/support/pickle.rb:11]: This is basically what I did above. I require `factory_girl_rails` instead of `factory_girl`, as this seems to know where to look for the factories on its own.

File: features/admin_manages_rotations.feature (hunk 2 above)
Comment (hermannloose) [inline features/admin_manages_rotations.feature:15]: You mean the three examples below or the three scenarios in total?

Regarding the examples: they are not really needed, as there aren't any real edge cases I'd like to check for yet—I don't want to test the error message for blank names right now, as they're all defaults from the scaffold and might change pretty soon—so I can take them out again if you think that's cleaner. They're more of a reminder for me to flesh things out in the future.

Comment (jupp0r) [inline features/admin_manages_rotations.feature:15]: Actually both. Also asking myself if those examles cases should be handled by one scenario or have one scenario each. Can you load those examples from some common file shared by some features/scenarios?

File: features/support/factory_girl.rb (hunk 1 above)
Comment (jupp0r) [inline features/support/factory_girl.rb:1]: Ok, makes sense.

File: features/support/pickle.rb (hunk 3 above)
Comment (jupp0r) [inline features/support/pickle.rb:11]: Ok.

File: features/admin_manages_rotations.feature (hunk 2 above)
Comment (hermannloose) [inline features/admin_manages_rotations.feature:15]: It should be three separate scenarios because they way either of those is handled might change separately.

Because the examples don't differ in their behaviour yet (again: more of a placeholder than real cases), they are all part of the same scenario. I'm hesitant to put them in a common file though, same reason as above: even though they now look the same in different places, I anticipate a lot of things to change in the near future—right now, constraints on escalation policy names and rotation names are the same, but that's more of an incidental detail—and I don't want to de-duplicate things prematurely.

Comment (jupp0r) [inline features/admin_manages_rotations.feature:15]: Alright then!
